ci-operator/step-registry/aws/c2s/mirror-images/aws-c2s-mirror-images-ref.yaml

```diff
@@ -0,0 +1,21 @@
+ref:
+  as: aws-c2s-mirror-images
```

Comment:
Why not reuse `ci-operator/step-registry/mirror-images/by-oc-adm/mirror-images-by-oc-adm-commands.sh`?

Reply:
they are different, `aws-c2s-mirror-images` mirrors `c2s-instance-metadata` and `cap-token-refresh` images, they are for C2S/SC2S only.

`mirror-images-by-oc-adm` mirrors OCP images.